city_scrapers/spiders/il_medicaid.py

```diff
@@ -0,0 +1,95 @@
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from datetime import datetime
+
+class IlMedicaidSpider(CityScrapersSpider):
+    name = "il_medicaid"
+    agency = "Illinois Medical Adivsory Committee"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.illinois.gov"]
+    # start_urls = ["file:///Users/ksong/myCodes/open-source/city-scrapers/tests/files/il_medicaid.html"]
+    start_urls = ["https://www.illinois.gov/hfs/About/BoardsandCommisions/MAC/Pages/"
+                  "MeetingSchedule.aspx"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        date_x_path = '//div[@id="ctl00_PlaceHolderMain_ctl01_ctl01__ControlWrapper_RichHtmlField"]/ul/li/p/text()'
+        for raw_date in response.xpath(date_x_path).getall():
+            # Clean up the raw dates
+            raw_date = raw_date.replace('\xa0',' ').strip()
+            if raw_date.count(',') == 2:
+                index = raw_date.find(",")
+                raw_date = raw_date[index+1:].strip()
+            date = datetime.strptime(raw_date, "%B %d, %Y")
+
+            meeting = Meeting(
+                title='Medicare Advisory Committee',
+                description='',
+                classification=ADVISORY_COMMITTEE,
+                start=date.replace(hour=10),
+                end=date.replace(hour=12),
+                all_day=False,
+                time_notes="",
+                location={
+                    'name': 'James R. Thompson Center',
+                    'address': '100 W Randolph St, 2nd flr. Rm. 2025, Chicago, IL 60601',
+                },  # Vid conf with Illinois Department of Healthcare and
+                    # Family Service in Springfield.
+                links=[],
```

Comment:
We're trying to prioritize pulling agendas and minutes when agencies post them, and it looks like they provide minutes on this page https://www.illinois.gov/hfs/About/BoardsandCommisions/MAC/Pages/MeetingMinutes.aspx and agendas/notices here https://www.illinois.gov/hfs/About/BoardsandCommisions/MAC/News/Pages/default.aspx

The way we've typically handled this in the past is scrape multiple pages, but only yield results on the last page after stitching back agendas and minutes to minutes by looking at the dates and names. You can see a more involved example for Detroit here: https://github.com/City-Bureau/city-scrapers/blob/master/city_scrapers/mixins/det_city.py Happy to talk more about this since it isn't always intuitive at first

Reply:
My apologies--I completely glossed over that part of the contribution guide. 

Thank you for the Detroit example!

Reply:
I am scraping the subcommittee pages, but I still need to work on getting the agendas and minutes.